style-sheets/primevue-components-overrides.scss

```diff
@@ -1140,4 +1140,133 @@ label {
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:nth-last-child(2)
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-top-right-radius: 0.25rem;
+    border-top-left-radius: 0;
+    border-right: solid 0.0625rem map.get(lightTheme.$light-theme, 'border-subtle-00');
+    border-left: none;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header.#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    font-weight: 700;
+    border-top: solid 0.125rem map.get(
+        lightTheme.$light-theme,
+        'border-interactive'
+    ) !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:last-child.#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-left: 0.0625rem solid map.get(lightTheme.$light-theme, 'text-secondary') !important;
+    border-right: 0.0625rem solid map.get(lightTheme.$light-theme, 'text-secondary') !important;
+    background-color: blue($color: #000000) !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:not(:first-child):not(:nth-last-child(2)).#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link{
+    border-right: 0.0625rem solid map.get(lightTheme.$light-theme, 'border-strong-01') !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:not(:first-child).#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-left: solid 0.0625rem map.get(lightTheme.$light-theme, 'border-strong-01');
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:not(.#{variables.$primevue-prefix}-highlight)
+.#{variables.$primevue-prefix}-tabview-nav-link:hover {
+    background-color: map.get(lightTheme.$light-theme, 'layer-accent-hover-01') !important;
+    border-color: map.get(lightTheme.$light-theme, 'layer-accent-hover-01') !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header
+.#{variables.$primevue-prefix}-tabview-nav-link:focus {
+    box-shadow: none !important;
+    outline: 0.125rem solid map.get(
+             lightTheme.$light-theme,
+             'border-interactive'
+         ) !important;
+    outline-offset: -0.12rem !important;
+}
+
+@media (min-width: 500px) {
+    .#{variables.$primevue-prefix}-tabview
+    .#{variables.$primevue-prefix}-tabview-nav-link {
+        min-width: 14.5rem !important;
+    }
+}
+
+@media (max-width: 390px) {
```

Comment:
did you confirm with Alison or Olga that we'll hide the tab on small screen?

Reply:
yes, Alison said that for now it will not be visible on small screens